packages/widget-chat/src/modules/redux-spark/component.js

```diff
@@ -39,9 +49,10 @@ class SparkComponent extends Component {
       connecting,
       authenticated,
       registered,
-      registering,
-      spark
-    } = nextProps;
+      registering
+    } = nextProps.spark.get(`status`).toJS();
+    const spark = nextProps.spark.get(`spark`);
```

Comment:
This is because spark is now an immutable object? is that why we have to call `spark.get`?